core: adding 'radosgw-admin' command support (#296)
rook/kubectl-rook-ceph: docs/radosgw-admin.md

# radosgw-admin

> This used to run any radosgw-admin cli command with with arbitrary args.

Reviewer: ```suggestion
This runs any radosgw-admin CLI command with arbitrary args.
```

## Examples

```bash
kubectl rook-ceph radosgw-admin user create --display-name="my user" --uid=myuser

> # Info: running 'radosgw-admin' command with args: [user create --display-name=johnny rotten --uid=johnny]

Reviewer: The output still has jonny rotten :)

Author: I just changed the name in the command missed updating the output

# {
#     "user_id": "johnny",
#     "display_name": "johnny rotten",

> #     "email": "",

Reviewer: I'd suggest we keep the output even simpler and remove lines 14-30.

> # Radosgw-admin

Reviewer: ```suggestion
# radosgw-admin
```

This used to run any radosgw-admin cli command with with arbitrary args.

## Examples

```bash

> kubectl rook-ceph radosgw-admin user create --display-name="johnny rotten" --uid=johnny

Reviewer: rotten? This suggestion might be boring though :)
```suggestion
kubectl rook-ceph radosgw-admin user create --display-name="my user" --uid=myuser
```

# Info: running 'radosgw-admin' command with args: [user create --display-name=johnny rotten --uid=johnny]

> # {

Reviewer: How about not include the full example output so we don't have to maintain it?
```suggestion
...
```
